### DISCUSSION TITLE
Proximity-based search

### DISCUSSION THREAD

**Discussion by krisk** [krisk/Fuse#660] [Ideas]
Should Fuse support [Proximity-based search](https://en.wikipedia.org/wiki/Proximity_search_(text))?

Let's see how many responses I get on the [tweet](https://twitter.com/kirorisk/status/1522424482819809280) 😄

**Comment by jeka57** [comment]
@krisk So what is a result, will it be implemented? :)

**Comment by krisk** [comment]
It's been on my mind, but I haven't gotten to it yet. Still something I want to explore — likely as an opt-in feature or plugin to keep the core lean. No timeline to share right now, but it's not forgotten.
